site/jms-client.md

```diff
@@ -367,6 +367,164 @@ may have configuration files or command-line options.
 Refer to the documentation for the target logging framework
 for configuration details.
 
+## <a id="rpc-support" class="anchor" href="#rpc-support">Support for Request/Reply (a.k.a. RPC)</a>
+
+It is possible to use JMS for synchronous request/reply use cases.
+This pattern is commonly known as *Remote Procedure Call* or *RPC*.
+
+### With JMS API
+
+An RPC client can be implemented in pure JMS like the following:
+
+<pre class="sourcecode java">
+Message request = ... // create the request message
+// set up reply-to queue and start listening on it
+Destination replyQueue = session.createTemporaryQueue();
+message.setJMSReplyTo(replyQueue);
+MessageConsumer responseConsumer = session.createConsumer(replyQueue);
+BlockingQueue&lt;Message&gt; queue = new ArrayBlockingQueue&lt;&gt;(1);
+responseConsumer.setMessageListener(msg -> queue.add(msg));
+// send request message
+MessageProducer producer = session.createProducer("request.queue");
+producer.send(request);
+// wait response for 5 seconds
+Message response = queue.poll(5, TimeUnit.SECONDS);
+// close the response consumer
+responseConsumer.close();
+</pre>
+
+Note this sample uses a `MessageListener` and a `BlockingQueue` to wait
+for the response. This implies a network roundtrip to register an AMQP
+consumer and another one to close the consumer.
+`MessageConsumer#receive` could have been used as well, in this case the JMS
```

Comment:
Is receive() advisable in this release?  If it has the 100ms delay issue that would be worth noting here.

Reply:
Delay issue mentioned.